<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="puzzle">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>601</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="puzzleGrid">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time: 0:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="movesLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Moves: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameIdLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Game:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <action name="Easy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
  </action>
  <action name="Medium">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
  </action>
  <action name="Hard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hard</string>
   </property>
  </action>
  <action name="actionNewGame">
   <property name="text">
    <string>New game</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
